precompiles/crowdloan-rewards/src/lib.rs

```diff
@@ -142,90 +92,73 @@ where
 {
 	// The accessors are first. They directly return their result.
 	fn is_contributor(
-		mut input: InputReader,
+		mut input: EvmDataReader,
 		target_gas: Option<u64>,
-	) -> Result<PrecompileOutput, ExitError> {
+	) -> EvmResult<PrecompileOutput> {
+		let mut gasometer = Gasometer::new(target_gas);
+
 		// Bound check
 		input.expect_arguments(1)?;
 
 		// parse the address
-		let contributor = input.read_address()?;
+		let contributor: H160 = input.read::<Address>()?.into();
+
+		let account: Runtime::AccountId = contributor.into();
```

Comment:
https://github.com/paritytech/frontier/blob/master/frame/evm/src/lib.rs#L195-L197

I think this is the official way to convert an H160 into the Runtime's `AccountId` type. Using this will also allow us to remove the trait bound `Runtime::AccountId: From<H160>`.